testing/unittests/unittests.cpp

```diff
@@ -186,6 +186,7 @@ int main(int argc, const char *argv[])
         excludeNames.append("*stress*");
         excludeNames.append("*timing*");
         excludeNames.append("*slow*");
+        excludeNames.append("CSysInfoLoggerTester"); // disabled by default as dali not available when executed within OBT
```

Comment:
Any unittests requiring dali access doesn't work when running within OBT.  I have disabled CSysInfoLoggerTester by default for this reason.  

I've created a jira to look into this: https://hpccsystems.atlassian.net/browse/HPCC-31955